tests/phpunit/integration/Modules/AdsTest.php

```diff
@@ -21,46 +23,97 @@
  */
 class AdsTest extends TestCase {
```

Comment:
Regarding the comment above about the feature flag, it might make more sense to create a new test case for all the Ads PAX (feature flag controlled) stuff and then enable that in the `set_up` so that it applies to all the tests, otherwise it needs to be enabled in each test it's used in here since we wouldn't want to blanket-apply it on the entire test case here.

Reply:
Added. There is one weird issue in new test, for some reason I can't get user to pass `VIEW_AUTHENTICATED_DASHBOARD` check, included authentication class, set the token, etc. Even tried forcing it with a filter. Do you have an idea?